bin/dune

```diff
@@ -23,6 +23,11 @@
  (files
   (dune.exe as dune)))
 
+(install
+  (section lib_root)
+  (package dune)
+  (files (dune.mlt as toplevel/dune)))
```

Comment:
Just something I realised since the initial design: if we call this file just `dune`, then when you are in a directory with a `dune` file and you do `#use "dune"` it's going to load the local `dune` file :/

Let's rename it to something else. I propose to rename it `dune.top` or `topdune`.

Reply:
Ah, okay. I'm not sure how to choose but I like `dune.top` better, so let's use that for now.

Reply:
Sounds good